cdc/owner/feed_state_manager.go

```diff
@@ -147,14 +177,18 @@ func (m *feedStateManager) handleAdminJob() (jobsPending bool) {
 		m.shouldBeRunning = true
 		jobsPending = true
 		m.patchState(model.StateNormal)
-		// remove error history to make sure the changefeed can running in next tick
+		// to make sure the changefeed can running in next tick,
+		// we remove the error history and reset the exponential backoff
 		m.state.PatchInfo(func(info *model.ChangeFeedInfo) (*model.ChangeFeedInfo, bool, error) {
 			if info == nil {
 				return nil, false, nil
 			}
 			if info.Error != nil || len(info.ErrorHis) != 0 {
 				info.Error = nil
 				info.ErrorHis = nil
+				m.lastErrorTime = time.Unix(0, 0)
+				m.expBackoff.Reset()
+				m.backoffInterval = m.expBackoff.NextBackOff()
```

Comment:
What is the reason for these three lines to be contained in the `PatchInfo` closure? What would happen if they were executed directly?

Reply:
> What is the reason for these three lines to be contained in the `PatchInfo` closure? What would happen if they were executed directly?

These lines can be moved out. I will fix it later.